frontend/src/app/general/breadcrumbs/breadcrumbs.component.html

```diff
@@ -5,6 +5,8 @@
 
 <ul>
```

Comment:
I'd add a `/` after `<ul>`, which looks like: 

![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/8936bd77-8248-4e34-b0b4-317abd7665ad)

This indicates that we have an "absolute" path from the root of the platform and not a relative path.